sass/routes/_admin-console.scss

```diff
@@ -827,12 +827,6 @@
         }
     }
 
-    .manage-teams__profile-picture {
-        border-radius: 20px;
-        height: 40px;
```

Comment:
@michaelgamble I converted this to 'lg' (36px) because we are already have 32px and 36px size.